staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -53,4 +59,5 @@ func init() {
 var defaultKubernetesFeatureGates = map[utilfeature.Feature]utilfeature.FeatureSpec{
 	StreamingProxyRedirects: {Default: true, PreRelease: utilfeature.Beta},
 	AdvancedAuditing:        {Default: false, PreRelease: utilfeature.Alpha},
+	APIResponseCompression:  {Default: false, PreRelease: utilfeature.GA},
```

Comment:
until we're sure we want this defaulting on, and as this is the first release it's included, I'd mark this Alpha...